slides/lesson1.md

```diff
@@ -797,6 +793,55 @@ x = x + 15;
 console.log(x); // what is the output here ?
 
 ```
+
+---
+
+### Warm-up
+
+A theater only lets people in that have a ticket or are on the guest list.
+
+What operator do we need to put instead of the `???`?
+
+```js
+let canEnterTheater = hasTicket ??? isOnGuestList;
+console.log("Person can enter theater: " + canEnterTheater);
+```
+
+---
+
+### Warm-up - Solution
+
+A theater only lets people in that have a ticket or are on the guest list.
+
+What operator do we need to put instead of the `???`?
+
+```js
+let canEnterTheater = hasTicket || isOnGuestList;
+console.log("Person can enter theater: " + canEnterTheater);
+```
+
+---
+
+### Warm-up 2
+
+A course only accepts people that paid the fee and passed the test
+
+```js
+let canJoinCourse = didPayFee ??? didPassTest;
+console.log("Person can join course: " + canJoinCourse);
+```
+
+---
+
+### Warm-up 2 - Solution
+
+A course only accepts people that paid the fee and passed the test
+
+```js
+let canJoinCourse = didPayFee && didPassTest;
+console.log("Person can join course: " + canJoinCourse);
+```
+
 ---
 
 ### Pass-by-value
```

Comment:
In the slides it says Primitive types in JavaScript (number, boolean, string, undefined) are always passed by value:
Maybe saying pass by value we say value is copied to the new variable. Because we are not really passing any parameters. Also pass by value is not clear to new comers.